kubernetes/config/frontend.yaml

```diff
@@ -13,7 +13,6 @@ data:
       schedulers: {
         'remote-execution': { address: 'scheduler:8982' },
       },
-      verifyActionResultCompleteness: true,
```

Comment:
This functionality should now be enabled by wrapping the AC configuration with a CompletenessCheckingBlobAccess.